integration-tests/kubernetes_multi_frontend.py

```diff
@@ -0,0 +1,158 @@
+# This integration test is taken straight fron `kubernetes_integration_test.py
+# With the setting 1 app, 1 model, 1 replica.
+# Adding multiple frontend checks (2 frontend)
+
+from __future__ import absolute_import, division, print_function
+import os
+import sys
+import requests
+import tempfile
+import shutil
+import json
+import numpy as np
+import time
+import logging
+import yaml
+from test_utils import (create_kubernetes_connection, BenchmarkException,
+                        fake_model_data, headers, log_clipper_state)
+
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, os.path.abspath("%s/../clipper_admin" % cur_dir))
+from clipper_admin import __version__ as clipper_version, CLIPPER_TEMP_DIR, ClipperException
+
+logging.basicConfig(
+    format='%(asctime)s %(levelname)-8s [%(filename)s:%(lineno)d] %(message)s',
+    datefmt='%y-%m-%d:%H:%M:%S',
+    level=logging.INFO)
+
+logger = logging.getLogger(__name__)
+MAX_RETRY = 5
+
+
+def deploy_model(clipper_conn, name, link=False):
+    app_name = "{}-app".format(name)
+    model_name = "{}-model".format(name)
+    clipper_conn.build_and_deploy_model(
+        model_name,
+        4,  # Version 4 to avoid container caching
```

Comment:
This seems brittle. Want to pick a random string instead? Versions can be arbitrary strings.